frontend/hub/execution-environments/ExecutionEnvironmentPage/hooks/useExecutionEnvironmentPageActions.tsx

```diff
@@ -28,6 +31,30 @@ export function useExecutionEnvironmentPageActions() {
           pageNavigate(HubRoute.EditExecutionEnvironment, { params: { id: ee?.name } }),
       },
       { type: PageActionType.Seperator },
+      {
+        type: PageActionType.Button,
+        selection: PageActionSelection.Single,
+        label: t('Sync from registry'),
+        onClick: (ee: ExecutionEnvironment) => {
+          const alert: AlertProps = {
+            variant: 'info',
+            title: t(`Sync started for remote registry "${ee.name}".`),
```

Comment:
these have different syntax here, `t` is a function so it doesn't see the string before substitution...

(the table in https://hackmd.io/sIX3koQ4SCSvhLRPYq8O9A#tldr-on-i18next-vs-lingui-differences  should help :))

Reply:
Good catch and that hackmd will come in handy, thanks :+1: